src/openrct2/rct2/S6Importer.cpp

```diff
@@ -262,11 +262,18 @@ class S6Importer final : public IParkImporter
 
         gResearchPriorities = _s6.active_research_types;
         gResearchProgressStage = _s6.research_progress_stage;
-        gResearchLastItem.rawValue = _s6.last_researched_item_subject;
+        if (_s6.last_researched_item_subject != RCT12_RESEARCHED_ITEMS_SEPARATOR)
+            gResearchLastItem = std::make_optional(
+                ResearchItem{ _s6.last_researched_item_subject, RESEARCH_CATEGORY_TRANSPORT });
```

Comment:
If you make a proper constructor for `ResearchItem`, then I think you can just do:

```
std::make_optional<ResearchItem>(_s6.last_researched_item_subject, RESEARCH_CATEGORY_TRANSPORT);
```